verticapy/_utils/_map.py

```diff
@@ -74,3 +74,49 @@ def verticapy_agg_name(key: str, method: Optional[Literal["vertica"]] = "") -> s
         elif key == "std":
             key = "stddev"
     return key
+
```

Comment:
@oualib
This is one option for using decorators to reuse parameter descriptions. In this case, you'd create a dictionary of parameter descriptions for each file (see classification.py) and then supply that dictionary and the parameters of the function to the decorator. The decorator would then insert the parameter descriptions between the function description and the listed return value.  
So, you would only need to define each parameter once in the parameter dictionary. Then all you need to do is supply the necessary parameter keys to the decorator.
To use the decorator, you'd just need to import the function add_docstring (didn't where this should go, so put it in _utils._map.py for now)
I don't know how this impacts function runtime, or how it interacts with nested decorators. 
This is my best shot for now. If this is a bad approach, just let me know and I'll review the doc as normal and come back to this sometime in the future.